client/src/TenderList.js

```diff
@@ -81,11 +81,7 @@ const TendersList = () => {
 								{tender.status}
 							</span>
 						</p>
-						<h2 className="title">
-							<a className="tender-id" href={`./bidding/${tender.id}`}>
-								{tender.title}
-							</a>
-						</h2>
+						<h2 className="title">{tender.title}</h2>
```

Comment:
Why did you remove the link?

Reply:
Because only buyer will be able to see the bids not all users. I have that link in buyer tender list.